packages/react-query-4/src/QueryErrorBoundary.tsx

```diff
@@ -4,7 +4,8 @@ import { type ComponentPropsWithoutRef, type ComponentRef, forwardRef } from 're
 
 /**
  * This component wrapping QueryErrorResetBoundary of `@tanstack/react-query` with `@suspensive/react`'s ErrorBoundary. So you must install `@suspensive/react` first, then use it. with this component, You don't have to make unnecessary repetitive implementation to combine ErrorBoundary with QueryErrorResetBoundary
- * @see {@link https://suspensive.org/docs/react-query/QueryErrorBoundary}
+ *
+ * @see {@link https://suspensive.org/docs/react-query/QueryErrorBoundary Suspensive Docs}
```

Comment:
```suggestion
 * @see {@link https://suspensive.org/en/docs/react-query/QueryErrorBoundary Suspensive Docs}
```

https://suspensive.org/docs/react-query/QueryErrorBoundary will be redirected to https://suspensive.org/en/docs/react-query/QueryErrorBoundary